tests/e2e/e2e_upgrade_handler_test.go

```diff
@@ -9,4 +11,7 @@ func (suite *IntegrationTestSuite) TestUpgradeHandler() {
 	suite.SkipIfUpgradeDisabled()
 	fmt.Println("An upgrade has run!")
 	suite.True(true)
+
+	// beforeUpgradeCtx := util.CtxAtHeight(suite.UpgradeHeight - 1)
```

Comment:
can be removed?

Reply:
left it for future upgrade handler e2e help. but can remove

Reply:
instead, i'll add a comment that tells future devs what to do with them